src/components/breadcrumb-item/breadcrumb-item.ts

```diff
@@ -62,7 +68,7 @@ export default class SlBreadcrumbItem extends LitElement {
                 class="breadcrumb-item__label breadcrumb-item__label--link"
                 href="${this.href}"
                 target="${this.target}"
-                rel=${ifDefined(this.target ? 'noreferrer noopener' : undefined)}
+                rel=${ifDefined(this.target ? rel : undefined)}
```

Comment:
The "target" could potentially be wrapped in an `ifDefined` but not sure if that's over engineering. Currently it results in a `target=""` when not supplied, but that still passes AXE apparently.